sympy/combinatorics/rewritingsystem_fsm.py

```diff
@@ -51,19 +45,17 @@ def __init__(self, name):
         self.name = name
         self.states = {} # Contains all the states in the machine.
 
-    def add_state(self, state_name, is_start=False, is_dead=False, is_accept=False, subst=None):
+    def add_state(self, state_name, state_type=None, rh_rule=None):
         '''
         Instantiate a state object and stores it in the 'states' dictionary.
 
         Arguments:
             state_name (instance of FreeGroupElement or string) -- name of the new states.
-            is_start (boolean) -- To mark if the state is a start state.
-            is_dead (boolean) -- To mark if the state is a dead state.
-            is_accept (boolean) -- To mark if the state is an accept state.
-            subt (instance of FreeGroupElement) -- right hand rule for dead state.
+            state_type (string) -- Denotes the type (accept/start/dead) of the state added.
```

Comment:
I think elaborating a bit to say that it's 'accept'/'start'/'dead' or 'a'/'s'/'d' respectively could be helpful to the user.

Reply:
Okay 👍